### PR TITLE
SNAP-2661 : Provide Snappy UI User a control over Auto Update

### PR DESCRIPTION
**Pull request by snappy-sachin**
## What changes were proposed in this pull request?

Requirement: 
  - Provide Snappy UI user a control using which he/she can start/stop the auto update of dashboard statistics.

Changes:
 - Adding JavaScript and CSS code changes for Auto Update On/OFF Switch on UI.

## How was this patch tested?

 - NA
 
Please review http://spark.apache.org/contributing.html before opening a pull request.

### CODE REVIEW

File: core/src/main/resources/org/apache/spark/ui/static/snappydata/snappy-memberdetails.js [hunk 1]
```diff
@@ -310,11 +321,12 @@ $(document).ready(function() {
...
-      // todo: need to provision when to stop and start update feature
-      // clearInterval(memberStatsUpdateInterval);
+  $("#myonoffswitch").on( 'change', toggleAutoUpdateSwitch );
 
+  var memberStatsUpdateInterval = setInterval(function() {
+    if(isAutoUpdateTurnedON) {
       loadMemberInfo();
-    }, 5000);
+    }
+  }, 5000);
```
Comment (sumwale) [inline core/src/main/resources/org/apache/spark/ui/static/snappydata/snappy-memberdetails.js:330]: What does this 5000 do here?

**Review by sumwale — COMMENTED** [review]
One query. Changes look good to me if we are ok with how the UI looks. Have asked Jags to provide his feedback.

File: core/src/main/resources/org/apache/spark/ui/static/snappydata/snappy-memberdetails.js (hunk 1 above)
Comment (snappy-sachin) [reply core/src/main/resources/org/apache/spark/ui/static/snappydata/snappy-memberdetails.js:330]: it is a 5 seconds (i.e. 5000 ms) timer for auto refresh.

**Comment by snappy-sachin** [comment]
As discussed in huddle (16 Nov 2018), merging the PR.

**Pull request merged**